plugins/disk/megaraid-controller-information

```diff
@@ -71,7 +71,7 @@ chomp $Parameters;
 my ($Adapter,$Type)=split(/-/,$Parameters);
 
 # Locate MegaCli64 application and remove whitespace
-my $Command=`which MegaCli64`;
+my $Command=`which MegaCli64`; if ( ! $Command ) { $Command=`which megacli`; }
```

Comment:
Thank you!

How about the following style?
```perl
my $Command=`which MegaCli64 megacli | head -n 1`;
```

I can imagine, that it is easier to read and extend ...

Reply:
Perfect!

Reply:
I am happy to merge, if your force-push this over your commit ...

Reply:
I have added the change...